zavod/zavod/logic/pep.py

```diff
@@ -0,0 +1,71 @@
+from enum import Enum
+from typing import Optional
+from datetime import datetime
+
+from zavod.context import Context
+from zavod import settings
+from zavod import helpers as h
```

Comment:
I don't think this loop is nice - and I don't think anything else is using backdate so I think it's best moved here til something else actually needs it.

Reply:
What would be needed for us to be able to make different cut-off dates for subnational and national PEPs in the `wd_peps` importer? I'd love to go 20 years on national and international peps again.

Reply:
one more argument - scope. I was tempted to tackle that in this PR thought I'd ship this first and then do it.

Reply:
@pudo would you keep any national PEP from any source for 20 years? My thinking was basically that any source that's confident about the scope of the position can pass it along, and any PEP where we're not sure about the scope is only kept according to the current rules.